transport-native-epoll/pom.xml

```diff
@@ -160,6 +160,7 @@
                        under 'META-INF/native' rather than 'META-INF/native/${platform}'. -->
                   <platform>.</platform>
                   <configureArgs>
+                    <arg>CC=clang</arg>
```

Comment:
We should not mandate clang. People can specify this as an environment variable before building.